style: footer border

restorenode opened the pull request.

restorenode committed:
style: footer border

JoshuaBrigati commented on src/styles/footer.scss:
```diff
@@ -9,6 +8,8 @@
   .links {
     display: flex;
     justify-content: space-between;
+    border-top: 1px solid rgb(202, 202, 202);
```
```suggestion
    border-top: 1px solid #ebeff8;
```

restorenode committed:
style: aside menu border

Changing light mode border color.  Adding aside menu border color to match footer border color.
